ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_adaptive_virtual_topology.schema.yml

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  router_adaptive_virtual_topology:
+    type: dict
+    keys:
+      topology_role:
+        description: Role profile
+        type: str
+        valid_values:
```

Comment:
@ClausHolbechArista @gmuloc - Are you ok with the proposed schema of valid values.
The CLI looks like this. In the CLI template, I replace the "-" with a space " ".

```
topology role edge
topology role pathfinder
topology role transit region
topology role transit zone
```

Reply:
based on discussion valid values will be implement with a space